samples/python/winml_tutorial/pywinrt_output/pyrt/src/py.Windows.Data.Json.h

```diff
@@ -0,0 +1,70 @@
+// WARNING: Please don't edit this file. It was generated by Python/WinRT
+
+#pragma once
+
+#include "pybase.h"
+
+#if __has_include("py.Windows.Foundation.Collections.h")
+#include "py.Windows.Foundation.Collections.h"
+#endif
+
+#include <winrt/Windows.Data.Json.h>
+
+namespace py
+{
+    template<>
+    struct winrt_type<winrt::Windows::Data::Json::JsonArray>
```

Comment:
winrt_type could be simplified in a couple ways:

1. inherit python_type via a base_winrt_type
2. pull get_python_type out into a function template - we use this pattern a lot in cppwinrt

Reply:
FYI, many of these comments are related to the core projection rather than the sample. I am doing a bunch of overhaul to the projection (in progress: https://github.com/Microsoft/xlang/tree/devhawk/pyoverhaul) where I will consider these suggestions. However, I am not going to block the sample PR to address them.

Reply:
I've opened #217 as a draft PR for comments related to core projection. Please comment there for topics that aren't specific to the sample

Reply:
Note, for every PR that updates the Python projection, I create a related PR in https://github.com/devhawk/pywinrt-output so that reviewers can examine the generated code (with a diff from previous version!) Any comments made in that repo will be addressed with changes to the pywinrt tool in this repo before committing the PR.

Reply:
Comment discsussion moved to https://github.com/Microsoft/xlang/pull/217#issuecomment-471405608